benchmark/AmberFF_bench.jl

```diff
@@ -34,5 +34,9 @@ amber_suite["update!"] = @benchmarkable update!($a_ff)
 
 update!(a_ff)
 
-amber_suite["compute_energy!"] = @benchmarkable compute_energy!($a_ff)
-amber_suite["compute_forces!"] = @benchmarkable compute_forces!($a_ff)
+amber_suite["compute_energy"] = @benchmarkable compute_energy($a_ff)
+amber_suite["compute_forces"] = @benchmarkable compute_forces($a_ff)
```

Comment:
```suggestion
amber_suite["compute_energy!"] = @benchmarkable compute_energy($a_ff)
amber_suite["compute_forces!"] = @benchmarkable compute_forces($a_ff)
```
These should not be changed. `compute_forces!` and `compute_energy!` are the correct names.